packages/block-editor/src/components/rich-text/style.scss

```diff
@@ -5,7 +5,6 @@
 }
 
 .editor-rich-text__editable {
-	margin: 0;
```

Comment:
@ellatrix can you recall why we added this rule? 

I removed it in this PR, because it bleeds into the `p` tag inside the Paragraph block, and as part of this PR we'd like to provide as clean and easily CSS overridable rules for that tag as possible. See https://github.com/WordPress/gutenberg/pull/14407/files#diff-7d328366ff8c0797fbdd334dcdf8b1a2R1

As far as I can tell, removing this one doesn't cause any visual regressions, but if you can recall why it's there it might help verify this.